tools/dreport.d/ibm.d/gendumpheader

```diff
@@ -50,6 +50,31 @@ function add_null() {
     printf '%*s' $a | tr ' ' "\0" >> $FILE
 }
 
+# Function to add Originator details to dump header
+function add_originator_details() {
+    if [ -z "$ORIGINATOR_TYPE" ]; then
```

Comment:
Are ORIGINATOR_TYPE and ORIGINATOR_ID mutually exclusive? 
I mean can there be a ORIGINATOR_ID when there is no ORIGINATOR_TYPE and viceversa?
If not , can we nest these as dependent, instead of having them independently?